packages/editor/src/store/utils/notice-builder.js

```diff
@@ -31,11 +31,18 @@ export function getNotificationArgumentsForSaveSuccess( data ) {
 	const publishStatus = [ 'publish', 'private', 'future' ];
 	const isPublished = includes( publishStatus, previousPost.status );
 	const willPublish = includes( publishStatus, post.status );
+	const isTrashed = post.status === 'trash';
 
 	let noticeMessage;
 	let shouldShowLink = get( postType, [ 'viewable' ], false );
 
-	if ( ! isPublished && ! willPublish ) {
+	// Since there is not a label for a post_type `trash` action,
+	// we add the message manually.
+	// Reference: https://developer.wordpress.org/reference/functions/get_post_type_labels/
+	if ( isTrashed ) {
+		noticeMessage = `${ postType.labels.singular_name } trashed.`;
```

Comment:
The `trashed` part of this string is not translatable.
Also, we should avoid translatable strings made by concatenating a variable part with other text as this is not well translatbale in some languages. This should use a _new_ post label to be introduced in core. Thus, it requires a patch for core. Note that in WordPress 5.0 new post labels were introduced for use in Gutenberg so this wouldn't be unprecedented.

Reply:
I opened a ticket for this: https://core.trac.wordpress.org/ticket/51387

Reply:
@ntsekouras a core ticket should not be a blocker for this.

Compat shims can be added to Gutenberg.

And even when that's not possible, we should not push strings that are not translatable.